lib/lroPollStrategy.ts

```diff
@@ -254,15 +254,28 @@ function getStatusFromResponse(response: HttpOperationResponse, responseBody?: a
   return result;
 }
 
-const terminalStates: LongRunningOperationStates[] = ["Succeeded", "Failed", "Canceled"];
+const terminalStates: LongRunningOperationStates[] = ["Succeeded", "Failed", "Canceled", "Cancelled"];
 
 /**
  * Get whether or not a long-running operation with the provided status is finished.
  * @param status The current status of a long-running operation.
  * @returns Whether or not a long-running operation with the provided status is finished.
  */
-function isFinished(status: LongRunningOperationStates): boolean {
-  return terminalStates.indexOf(status) !== -1;
+export function isFinished(status: LongRunningOperationStates): boolean {
+  let result = false;
+  for (const terminalState of terminalStates) {
+    if (longRunningOperationStatesEqual(status, terminalState)) {
+      result = true;
+      break;
+    }
+  }
+  return result;
+}
+
+export function longRunningOperationStatesEqual(lhs: LongRunningOperationStates, rhs: LongRunningOperationStates): boolean {
```

Comment:
What does `lhs` mean? Left...?

Reply:
Left Hand Side (of the expression). It's something I remember seeing a long time ago and it's stuck with me. I had the same reaction when I first saw it.